projects/kubeedge/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: "https://kubeedge.io/en/"
+primary_contact: "Adam@adalogics.com"
+auto_ccs :
+  - "David@adalogics.com"
```

Comment:
I think we want to put you guys in vendor_ccs from now on.
Right @oliverchang ?